docs/config.md

```diff
@@ -293,6 +293,34 @@ This option defines the request timeout value for sending requests retrieving a
 
 By default, this value is set to 4000 milliseconds.
 
+
+### Option "cf.request.backoff.initial" (optional)
```

Comment:
I imagine the `max` as being just for the jitter factor. Since if a user sets the backoff to `10,000ms`, and we hardcoded a `max` as `intial + 1,000ms` it is easy enough to reason about as 'backoff of X (with up to 1 second of jitter)

I'm not too set against including the `initial` and `max`,I mainly didn't want to bloat the documentation since the 2 options are similar enough to cause confusion to a typical user. (Similar to how the two cache refresh/expiry options are similar enough to cause confusion)

Reply:
mmmh. I get your point. 
I don't like this fixed value.
What about `this.maxBackoffDelay = Duration.ofMillis((long) (firstBackoffDelay.toMillis() * 1.5));`? That is to say, we always use the max jitter?

Reply:
So with initial 500ms  max is 750ms?  1s and 1.5s, 10s and 15s?  Sounds like a good idea

Reply:
yes. So, we have a deal 😁

Reply:
Adjusted with 94b52c193e7ed359f88a2f234238b8a02a65d557